build-scripts/cassandra-artifacts_s390x.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -xe
```

Comment:
it's unclear to me how this file is different to the existing `build-scripts/cassandra-artifacts.sh` ?

Reply:
Yes..deb and rpm packaging is removed for s390x for now as apache cassandra needs few code changes to fix test failures on s390x and we have already provided code changes through  https://issues.apache.org/jira/browse/CASSANDRA-17723 . Also, chronicle-bytes code used in Apache Cassandra needs s390x support hence we are going ahead with this workaround for now.

Reply:
Ok. Is the plan to fix those failures as part of 17723 or 18072 ? 

If that is the case then this PR should be marked as a draft, as the intention is never to merge it.

If you want to delay fixing the deb and rpm packing, then I suggest adding a conditional around that in the original script. Something like `DEP_PACKAGING_SKIP` and `RPM_PACKAGING_SKIP`.

Reply:
> Ok. Is the plan to fix those failures as part of 17723 or 18072 ?
> 
Issue-17723 will fix 6 tests out of 13 tests which we are skipping for s390x.
Issue-18072 is a part of CI for s390x.  I raised different issue as repository was different.

> If that is the case then this PR should be marked as a draft, as the intention is never to merge it.
> 
> If you want to delay fixing the deb and rpm packing, then I suggest adding a conditional around that in the original script. Something like `DEP_PACKAGING_SKIP` and `RPM_PACKAGING_SKIP`.
I think we can go ahead with .deb and .rpm packaging for s390x also. Lets not delay.